config/clouddriver.yml

```diff
@@ -101,7 +101,9 @@ dockerRegistry:
     - name: ${providers.dockerRegistry.primaryCredentials.name}
       address: ${providers.dockerRegistry.primaryCredentials.address}
       username: ${providers.dockerRegistry.primaryCredentials.username:}
-      passwordFile: ${providers.dockerRegistry.primaryCredentials.passwordFile}
+      passwordFile: ${providers.dockerRegistry.primaryCredentials.passwordFile:}
+      repositories:
+        - ${providers.dockerRegistry.primaryCredentials.repository}
```

Comment:
I omit this because I prefer to depend on the `/v2/_catalog` endpoint to retrieve all repos